sdk/auth/auth_test.go

```diff
@@ -183,6 +183,7 @@ func TestAccNewAuthorizerFromCredentials(t *testing.T) {
 
 	credentials := auth.Credentials{
 		AuxiliaryTenantIDs:            test.AuxiliaryTenantIds,
+		AzureCliSubscriptionIDHint:    test.SubscriptionId,
```

Comment:
for this one, do we only want to set this for the CLI test-case below? (207/208)?